source/average_py.f90

```diff
@@ -7,16 +7,25 @@ module AvIn
       logical InPostRotateSyst
```

Comment:
I see a few compiler warnings coming from this file:

Warning: Unused variable ‘chi2loc’ declared at (1) [-Wunused-variable]
data.inc:109:19: Warning: Unused variable ‘gridtype’ declared at (1) [-Wunused-variable]
average_py.f90:169:29:

Warning: Unused variable ‘idxsys’ declared at (1) [-Wunused-variable]
average_py.f90:189:39:

---> please clean up them